blockchain/tx_pool.go

```diff
@@ -870,13 +883,13 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (bool, error) {
 	// (2) remove an old Tx with the largest nonce from queue to make a room for a new Tx with missing nonce
 	// (3) discard a new Tx if the new Tx does not have a missing nonce
 	// (4) discard underpriced transactions
-	if uint64(len(pool.all)) >= pool.config.ExecSlotsAll+pool.config.NonExecSlotsAll {
+	if uint64(pool.all.Count()) >= pool.config.ExecSlotsAll+pool.config.NonExecSlotsAll {
```

Comment:
[Based on your slot concept](https://github.com/klaytn/klaytn/pull/1733#discussion_r1044050420), the `pool.all.Count()` should be changed to `pool.all.Slots() + num Slots(tx)` ? this is because the `pool.all.Count()` does not consider the amount of total tx size.

Reply:
You're right, i have updated to add Slots size. By the way, this PR is too complicated, so i divided into two PRs. Could you review the next PR first? https://github.com/klaytn/klaytn/pull/1748

Reply:
@yoomee1313 https://github.com/klaytn/klaytn/pull/1733#issuecomment-1367205381 You mean the #1748 need to be reviewed first?
I think It seems to be confused "Please review this PR first."

Reply:
@kjeom Oh, sorry. It is right that #1748 need to be reviewed first

Reply:
@hyunsooda would you check this?

Reply:
Got it. thansk.